src/VisualStudio/Core/Def/Implementation/Preview/FileChange.cs

```diff
@@ -210,7 +210,8 @@ public TextDocument GetUpdatedDocument()
             return _right.WithText(UpdateBufferText());
         }
 
-        public bool IsAdditionalDocumentChange => !((_left ?? _right) is Document);
+        public bool IsAdditionalDocumentChange => !((_left ?? _right) is Document) && !IsAnalyzerConfigDocumentChange;
+        public bool IsAnalyzerConfigDocumentChange => (_left ?? _right) is AnalyzerConfigDocument;
```

Comment:
ick, i don't like this implicit relationship.  basically, if something is an AdditionalDoc i'd like that to be explicit.  i don't want it to just fall out because it's "not an analyzer doc".  the latter form just means more complexity pushed to the edges if/when we ever add new doc types.

Reply:
Should be cleaner now with document kind check.